src/pages/workspaces/workspace/Dashboard/RightBoxSection.ts

```diff
@@ -27,7 +31,14 @@ export const RightBoxSection = (props: RightBoxSectionProps): ReactNode => {
           initialOpenState: panelOpen,
           titleFirst: true,
           afterTitle,
-          onClick: () => setPanelOpen(!panelOpen),
+          onOpenChanged: (panelOpen) => {
```

Comment:
This fixes a bug. The `onClick` handler was being called whenever anything within the Collapse was clicked, not just when the top-level button toggle was clicked. Thus doing things like clicking on the link to open the Google Project in another tab would cause `setPanelOpen` to be called here, which in turn persisted incorrect "opened" state. Thus if you refreshed the Workspace Dashboard, the RightBoxSection would not be correctly expanded/collapsed.

The fix is to register a handler with the `Collapse` instance to receive a notification when it actually expands/collapses.